src/preset_cli/cli/superset/sync/dbt/lib.py

```diff
@@ -386,20 +386,21 @@ def apply_select(
     """
     Apply dbt node selection (https://docs.getdbt.com/reference/node-selection/syntax).
     """
-    if not select:
-        return models
```

Comment:
Also, this was wrong. We should only return all models if `exclude` is also falsy.